presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -62,20 +59,18 @@ public Connector create(String catalogName, Map<String, String> requiredConfig,
     {
         requireNonNull(requiredConfig, "requiredConfig is null");
 
-        try (ThreadContextClassLoader ignored = new ThreadContextClassLoader(classLoader)) {
-            Bootstrap app = new Bootstrap(
-                    binder -> binder.bind(TypeManager.class).toInstance(context.getTypeManager()),
-                    new JdbcModule(catalogName),
-                    new CredentialProviderModule(),
-                    module);
+        Bootstrap app = new Bootstrap(
+                binder -> binder.bind(TypeManager.class).toInstance(context.getTypeManager()),
+                new JdbcModule(catalogName),
+                new CredentialProviderModule(),
+                module);
```

Comment:
In theory the necessity of setting TCCL may depend on the actual `module` passed here.

Reply:
Nope, the engine sets this when calling create.